sql/telemetry_derived/clients_histogram_aggregates_v1/query.sql

```diff
@@ -9,15 +43,21 @@ CREATE TEMP FUNCTION udf_merged_user_data(old_aggs ANY TYPE, new_aggs ANY TYPE)
     key STRING,
     process STRING,
     agg_type STRING,
-    aggregates ARRAY<STRUCT<key STRING, value INT64>>>> AS (
+    old_aggregates ARRAY<STRUCT<key STRING, value INT64>>,
+    new_aggregates ARRAY<STRUCT<key STRING, value INT64>>>> AS (
   (
     WITH unnested AS
-      (SELECT *
+      (SELECT
+        * EXCEPT(old_aggregates),
+        new_aggregates AS old_aggregates
       FROM UNNEST(old_aggs)
 
       UNION ALL
 
-      SELECT *
+      SELECT
+        * EXCEPT(aggregates),
```

Comment:
Where is `aggregates` defined in the `histogram_aggregates` struct (which should be the same type as `new_aggs`).